nodejs/BUILD

```diff
@@ -105,6 +108,61 @@ deploy_npm(
     release = deployment["npm.release"],
 )
 
+ts_project(
+    name = "typedoc_runner",
+    srcs = glob([
+        "tool/typedoc/TypeDocRunner.ts",
+    ]),
+    tsconfig = "tool/typedoc/tsconfig.json",
+    declaration = True,
+    deps = [
+        ":node_modules/typedoc",
+        ":node_modules/@types/node",
+    ],
+    visibility = ["//visibility:public"],
+    out_dir = ".",
+)
+
+js_binary(
+    name = "typedoc_runner_bin",
+    data = [
+        ":typedoc_runner",
+    ],
+    entry_point = ":typedoc_runner",
+    chdir = "nodejs",
+)
+
+typedoc_docs(
+    name = "driver_docs",
```

Comment:
I don't think we need a prefix of `driver_` for these. I think we can just call this `docs_html` and the parsed one `docs_adoc`. We can apply this to all our languages that have this split pattern of html/adoc rules